internal/olm/operator/registry/fbcindex/fbc_registry_pod.go

```diff
@@ -194,13 +193,38 @@ func (f *FBCRegistryPod) podForBundleRegistry() (*corev1.Pod, error) {
 		return nil, err
 	}
 
+	// create ConfigMap if it does not exist,
+	// if it exists, then update it with new content.
+	cm, err := f.createConfigMap(cs)
+	if err != nil {
+		return nil, fmt.Errorf("configMap error: %w", err)
+	}
+
 	// make the pod definition
 	f.pod = &corev1.Pod{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      getPodName(bundleImage),
 			Namespace: f.cfg.Namespace,
 		},
 		Spec: corev1.PodSpec{
+			Volumes: []corev1.Volume{
+				{
+					Name: k8sutil.TrimDNS1123Label(cm.Name + "-volume"),
+					VolumeSource: corev1.VolumeSource{
+						ConfigMap: &corev1.ConfigMapVolumeSource{
+							Items: []corev1.KeyToPath{
+								{
+									Key:  "extraFBC",
+									Path: path.Join(defaultConfigMapName, "extraFBC.yaml"),
```

Comment:
Looks like `extraFBC` is used in 4 different places, we should make it a constant and reuse that throughout the file.

Reply:
Made it a constant.